### ISSUE TITLE
Overwatch ban

### ISSUE THREAD

**Issue by WorestWasTaken** [leo4048111/Potato-Injector#11]
does this work with overwatch bans?

**Comment by smthpy** [comment]
No, Overwatch consists of manual bans from the player community. This only applied to CS:GO, which is no longer functioning with official VAC-sanctioned servers.

OW is no longer present in CS2.

**Comment by leo4048111** [comment]
Not really sir...
